projects/storefrontlib/src/lib/ui/pages/cart-page/cart-page.component.html

```diff
@@ -1 +1,8 @@
-<cx-cart-page-layout></cx-cart-page-layout>
+<cx-page-layout
+  *ngIf="(cart$ | async) as cart"
+  [class.empty]="!cart.totalItems"
+>
+  <ng-container *ngIf="cart.totalItems > 0">
```

Comment:
If possible, it would be good to move this logic to the CMS component and remove CartPageComponent - just to eaisly get rid of Cart's route in the future.

Reply:
Done as much as possible now.